### ISSUE TITLE
KeyCloak chart 9.2.11 references old config-cli image which does not contain KeyCloak 18.0.1

### ISSUE THREAD

**Issue by markusheiden** [bitnami/charts#10796]
### Name and Version

bitnami/keycloak:9.2.11

### What steps will reproduce the bug?

1. Use config-cli job

### Are you using any custom parameters or values?

No

### What is the expected behavior?

The config-cli job runs successfully.

### What do you see instead?

`Error: Unable to access jarfile /opt/bitnami/keycloak-config-cli/keycloak-config-cli-18.0.1.jar`

### Additional information

The current chart references a config-cli image that does not contain KeyCloak 18.0.1. Because the chart forces the config-cli job to use the app version of the chart (which is 18.0.1) the config-cli job fails.

**Comment by markusheiden** [comment]
Workaround:
```yaml
keycloakConfigCli:
  command:
    - java
    - -jar
    - /opt/bitnami/keycloak-config-cli/keycloak-config-cli-18.0.0.jar
```

**Comment by javsalgar** [comment]
Hi,

Thank you so much for reporting. I will open a ticket to fix the issue. As soon as there are news, we will update the ticket.

**Comment by aoterolorenzo** [comment]
Hi @markusheiden,

I'm afraid there's no an optimal solution here. Since we are bundling `keycloak-config-cli` into the `keycloak` chart, we can't preview or sync directly between both artifacts releases, since we could encounter ourselves at the point of wanting to release a new version of the chart, and `keycloak-config-cli` still not being implementing that version (and not including support for generating that jar), _which is exactly the root of your issue_. It also could happen the opposite too.

That's why, we have synced our chart to point directly to the latest available jar version for `keycloak-config-cli`, so right now our latest chart of version 18.0.2 is pointing to the 19.0.1 jar (that is the opposite case I've mention). After do some testing we can say that is working fine, but as I said it won't be completely synced.

We have queued a task to release a new chart for `keycloak`, so hope that in the upcoming days we have a new chart using `keycloak` 19.0.1 pointing to the `keycloak-config-cli` jar version 19.0.1 (or at least to the same major).
